x/group/client/testutil/tx.go

```diff
@@ -559,7 +559,7 @@ func (s *IntegrationTestSuite) TestTxUpdateGroupMembers() {
 
 	validUpdatedMembersFileName := testutil.WriteToNewTempFile(s.T(), fmt.Sprintf(`{"members": [{
 		"address": "%s",
-		"weight": "0",
+		"weight": "1",
```

Comment:
an updated member weight can be 0, in case we want to remove the member from the group